source/MissionAction.h

```diff
@@ -88,6 +88,10 @@ class MissionAction {
 	int64_t payment = 0;
 	int64_t paymentMultiplier = 0;
 	
+	int64_t fine = 0;
+	int64_t fineTerm = -1;
```

Comment:
I think it's okay for us to define the default term and interest rates here. (Probably the term should just be an `int`.)

Would doing so reduce the branching needed for calling AddFine?

Reply:
The AddFine with an interest rate is a different function than the one without since the interest rate one has to calculate a credit score.

I'll merge the two AddFines functions.